# Description: don't use the unversioned python which is going away
# Upstream: needed
#
Index: libqmi-1.24.6/build-aux/qmi-codegen/qmi-codegen
===================================================================
--- libqmi-1.24.6.orig/build-aux/qmi-codegen/qmi-codegen
+++ libqmi-1.24.6/build-aux/qmi-codegen/qmi-codegen
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- Mode: python; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*-
 #
 # This program is free software; you can redistribute it and/or modify it under
